src/components/duo/Filter.tsx

```diff
@@ -144,7 +144,7 @@ const Filter = ({ disclosure, allOpen, toggleAll, requestParams, updateParams }:
 
   useEffect(() => {
     updateParams({ lanes: selected.join(',') });
-  }, [selected, updateParams, requestParams.lanes]);
+  }, [selected]);
```

Comment:
```suggestion
  }, [selected, updateParams]);
```
로 만들고 `src/pages/duo.tsx:53`에 있는 `updateRequestParams()`를 다음과 같이 업데이트 하죠.
```tsx
  const updateRequestParams = useCallback(
    (toUpdate: Record<string, DuoSummonersRequest[keyof DuoSummonersRequest]>) => {
      setRequestParams((prev) => ({ ...prev, ...toUpdate }));
    },
    [],
  );
```